ctx.go

```diff
@@ -1037,67 +1037,197 @@ func (c *DefaultCtx) Queries() map[string]string {
+		result, err := strconv.ParseInt(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int](int(result))
+	case int8:
+		result, err := strconv.ParseInt(q, 10, 8)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int8](int8(result))
+	case int16:
+		result, err := strconv.ParseInt(q, 10, 16)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int16](int16(result))
+	case int32:
+		result, err := strconv.ParseInt(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int32](int32(result))
+	case int64:
+		result, err := strconv.ParseInt(q, 10, 64)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, int64](result)
+	case uint:
+		result, err := strconv.ParseUint(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint](uint(result))
+	case uint8:
+		result, err := strconv.ParseUint(q, 10, 8)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint8](uint8(result))
+	case uint16:
+		result, err := strconv.ParseUint(q, 10, 16)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint16](uint16(result))
+	case uint32:
+		result, err := strconv.ParseUint(q, 10, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint32](uint32(result))
+	case uint64:
+		result, err := strconv.ParseUint(q, 10, 64)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, uint64](result)
+	case float32:
+		result, err := strconv.ParseFloat(q, 32)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, float32](float32(result))
+	case float64:
+		result, err := strconv.ParseFloat(q, 64)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, float64](result)
+	case bool:
+		result, err := strconv.ParseBool(q)
+		if err != nil {
+			if len(defaultValue) > 0 {
+				return defaultValue[0]
+			}
+			return v
+		}
+		return assertValueType[V, bool](result)
+	case string:
+		if q == "" && len(defaultValue) > 0 {
+			return defaultValue[0]
+		}
+		return assertValueType[V, string](q)
+	case []byte:
+		if q == "" && len(defaultValue) > 0 {
+			return defaultValue[0]
+		}
+		return assertValueType[V, []byte](utils.UnsafeBytes(q))
```

Comment:
can you use getBytes here

Reply:
@efectn like this:
![image](https://github.com/gofiber/fiber/assets/17839029/9309b6ff-5374-4b1e-890a-b97c717d4620)

Reply:
> @efectn like this: ![image](https://private-user-images.githubusercontent.com/17839029/296851620-9309b6ff-5374-4b1e-890a-b97c717d4620.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDUzNDYwNzUsIm5iZiI6MTcwNTM0NTc3NSwicGF0aCI6Ii8xNzgzOTAyOS8yOTY4NTE2MjAtOTMwOWI2ZmYtNTM3NC00YjFlLTg5MGEtYjk3YzcxN2Q0NjIwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE1VDE5MDkzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1MTQ3M2MwNmI2YzMzYTg5YzYwN2VmN2VmMDIzZjI2YTgwMjE3MjFjYWU5OTAyZjNiNjU4ZTU1NjE4ZGNiMGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Hu7UPGhnyAqXLl0b2Mq6ZfeoEBic8rLuxGDS99V3SAk)

`ctx.app.getBytes(q)`

Reply:
@efectn OK, I changed it and do I repost it for the benchmark?

Reply:
> @efectn OK, I changed it and do I repost it for the benchmark?

Not needed, thanks

Reply:
> > @efectn OK, I changed it and do I repost it for the benchmark?
> 
> Tidak dibutuhkan, terima kasih

I've pushed, please check.